torch/_dynamo/variables/nn_module.py

```diff
@@ -283,7 +283,8 @@ def call_method(
         if name == "__call__":
             # TODO(whc)  do we really need this special case?
             return self.call_function(tx, args, kwargs)
-        elif name == "forward":
+
+        elif name in ["forward", "_call_impl"]:
```

Comment:
I have a better way: adding ```_call_impl``` to L283, then you don't need to handle L299-L303. I think that's more concise and correct, though they are equivalent in functionality.

Reply:
In ```call_function```, we handle inlining ```__call__```(including hooks). However, in this ```call_method```, we only handle directly calling to ```forward``` and inline it(not including hooks).

Reply:
Seems like it was fine? I don't see any dynamo failures because of this change

Reply:
Yea, it's fine.